packages/commercetools/theme/components/MyAccount/BillingAddressForm.vue

```diff
@@ -77,6 +77,31 @@
             :errorMessage="errors[0]"
           />
         </ValidationProvider>
+        <ValidationProvider
+          :rules="`required|oneOf:${countries.map(c => c.name).join(',')}`"
```

Comment:
Let's create a rule out of it. It will be easier to understand the logic of a rule when it will be in script tag rather than in the template